### ISSUE TITLE
Test failure(s): core.thing.firmware.FirmwareUpdateServiceOSGiTest

### ISSUE THREAD

**Issue by maggu2810** [eclipse/smarthome#1992]
Hudson build failed.

```text
assert that update firmware handles general handler error(org.eclipse.smarthome.core.thing.firmware.FirmwareUpdateServiceOSGiTest)  Time elapsed: 11.459 sec  <<< FAILURE!
java.lang.AssertionError: 
Expected: is <1>
     but: was <3>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at org.junit.Assert.assertThat(Assert.java:832)
	at org.junit.Assert$assertThat$1.callStatic(Unknown Source)
	at org.eclipse.smarthome.core.thing.firmware.FirmwareUpdateServiceOSGiTest$_assertFirmwareStatusInfoEvent_closure42.doCall(FirmwareUpdateServiceOSGiTest.groovy:822)
	at org.eclipse.smarthome.core.thing.firmware.FirmwareUpdateServiceOSGiTest$_assertFirmwareStatusInfoEvent_closure42.doCall(FirmwareUpdateServiceOSGiTest.groovy)
	at sun.reflect.GeneratedMethodAccessor66.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:39)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at org.eclipse.smarthome.test.OSGiTest.waitForAssert(OSGiTest.groovy:181)
	at org.eclipse.smarthome.test.OSGiTest.waitForAssert(OSGiTest.groovy)
	at sun.reflect.GeneratedMethodAccessor26.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:133)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:141)
	at org.eclipse.smarthome.core.thing.firmware.FirmwareUpdateServiceOSGiTest.assertFirmwareStatusInfoEvent(FirmwareUpdateServiceOSGiTest.groovy:821)
	at org.eclipse.smarthome.core.thing.firmware.FirmwareUpdateServiceOSGiTest.this$3$assertFirmwareStatusInfoEvent(FirmwareUpdateServiceOSGiTest.groovy)
```

Full log: [SmartHomeDistribution-Nightly#1242.txt](https://github.com/eclipse/smarthome/files/413553/SmartHomeDistribution-Nightly.1242.txt)

**Comment by maggu2810** [comment]
@tomhoefer Could you have a look at (because it is raised by the firmware stuff)

**Comment by tomhoefer** [comment]
i will check

**Comment by maggu2810** [comment]
Should be solved by #2004
